### DISCUSSION TITLE
CLI app grant-access to a team

### DISCUSSION THREAD

**Discussion by jan-from-meiro** [envkey/envkey#105] [Q&A]
Is there any possibility to grant access to app to a team over CLI, not just UI?

envkey apps grant-access [app] [person] [app-role]

Person seems to be the only option here.

**Comment by danenania** [comment]
Hi @jan-from-meiro, sorry about the delayed response on this. This feature actually was shipped recently, but I didn't include it in an auto-upgrade since it was requested by a single customer and didn't seem worth triggering an auto-upgrade for everyone.

That said, if you install the latest version of the CLI (2.5.0), you will now have the option to grant app access to a team as you're describing: https://docs-v2.envkey.com/docs/cli-overview

**Comment by jan-from-meiro** [comment]
Hi @danenania , that's awesome! Didn't expect such a swift resolution of this. Thank you!

**Reply by danenania** [reply]
I'm happy to know it's useful to more than one customer 🙂
